api/common.go

```diff
@@ -15,11 +15,11 @@ import (
 	"github.com/docker/libtrust"
 )
 
+// Version of Current REST API
+var Version = version.Version("1.22")
```

Comment:
This is really for the CLI not the daemon, but to your other point about using a global... I actually would have preferred if we had proper getters and setters for this field but given the constant was already a package global I went with that pattern trying to minimize the amount of changes.